lib/pagination_search/search_conditions.rb

```diff
@@ -38,9 +38,9 @@ def wrapped(cleaned_grouped_hash)
       end
 
       def separated(search_string)
-        CSV::parse_line(search_string, col_sep: ' ')
+        CSV::parse_line(search_string, col_sep: ' ').compact
       rescue CSV::MalformedCSVError => mce
-        nil
+        CSV::parse_line(search_string.tr('"', ' '), col_sep: ' ').compact
```

Comment:
not the biggest fan of this approach which is using exception catching to control flow -- which is an anti-pattern